etna/datasets/tsdataset.py

```diff
@@ -635,8 +644,10 @@ def to_pandas(self, flatten: bool = False) -> pd.DataFrame:
         2021-06-05      1.00      1.00
         """
         if not flatten:
-            return self.df.copy()
-        return self.to_flatten(self.df)
+            if columns is None:
+                return self.df.copy()
+            return self.df.loc[:, self.idx[:, columns]].copy()
```

Comment:
We should investigate is it a problem to pass ":" here in place of segments.

The problem is described here: #775.

Reply:
I am not sure that it is important here, as this method just return the copy of dataframe with the specified columns

Reply:
If it leads to swapping values of columns (that was a mistake), then you will get a broken dataframe. Ask the author of that bug about it.

Reply:
Made it deterministic, add comment in https://github.com/tinkoff-ai/etna/issues/775 to fix this place either